nextjsUIapp/src/app/knowledge-base/[slug]/[id]/page.tsx

```diff
@@ -1,122 +1,104 @@
+                    }));
+                }
+            }
+        };
+
+        if (extractedBoxes.length > 0) {
+            fetchBoxData();
+        }
+    }, [extractedBoxes, id]);
 
     return (
         <div className="kb-page-margin">
-            <SideBarKBFromConfig/>
+            <SideBarKBFromConfig />
             <div className="grid fix-left-margin grid-cols-1">
                 <div className="w-full bg-white shadow-md rounded-lg overflow-hidden">
                     <div className="p-4">
-                        <h2 className="text-xl font-bold">{mainCardTitle}</h2>
+                        <h2 className="text-xl font-bold"> {decodeURIComponent(id).substring(decodeURIComponent(id).lastIndexOf("/") + 1)}</h2>
                         <p className="text-gray-700">{mainCardDescription}</p>
                     </div>
                     <div className="p-4 border-t border-gray-200 flex">
                         {extractedBoxes.map((entitycards, index) => (
-                            entitycards.cardtype === "summarybox" ? (
-                                <div key={index} className="text-gray-700 text-base space-x-4">
-                                    {entitycards.slug === "summarybox" ? (
-                                        <div className="w-full bg-white shadow-md rounded-lg overflow-hidden">
-                                            <div className="bg-gray-400 p-4">
-                                                <h2 className="text-white text-xl font-bold">{
-                                                    decodeURIComponent(id).substring(decodeURIComponent(id).lastIndexOf('/') + 1)
-                                                }
-                                                </h2>
-                                            </div>
-                                            <div className="p-4">
-                                                <p className="text-gray-700">{entitycards.description}</p>
-                                                <p className="text-gray-700">
-                                                    {fetchBoxes(entitycards.sparql_query?.replace(/\{0\}/g, decodeURIComponent(id)) || 'No query available')}
-                                                </p>
-                                            </div>
+                            <div key={index} className="text-gray-700 text-base space-x-4">
+                                {entitycards.cardtype === "card" && (
+                                    <div className="w-full bg-white shadow-md rounded-lg overflow-hidden">
+                                        <div className="bg-gray-400 p-4">
+                                            <h2 className="text-white text-xl font-bold">
+                                                {entitycards.name}
+                                                {/*{decodeURIComponent(id).substring(decodeURIComponent(id).lastIndexOf("/") + 1)}*/}
```

Comment:
Just flagging if you want to remove this commented-out code

Reply:
yes, removed.